Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -448,11 +448,19 @@ private void UpdateInvalidLoginAttemptCount(User user, int newValue)
 
             user.Policy.InvalidLoginAttemptCount = newValue;
 
-            var maxCount = user.Policy.IsAdministrator ? 3 : 5;
+            // Check for users without a value here and then fill in the default value
+            // also protect from an always lockout if misconfigured
+            if (user.Policy.LoginAttemptsBeforeLockout == null || user.Policy.LoginAttemptsBeforeLockout == 0)
```

Comment:
I feel like this value should've been set prior to reaching this

Reply:
So this is for users that currently exist. They should (theoretically) have a value of null here. This will keep the old expected behavior in play until their accounts get the setting updated. The only other place I could think to set this was in the load code for the Policies and I'd rather not do it there.